collector.go

```diff
@@ -201,6 +248,18 @@ func (c *spaceliftCollector) Collect(metricChannel chan<- prometheus.Metric) {
 	metricChannel <- prometheus.MustNewConstMetric(c.currentBillingPeriodUsedPublicSeconds, prometheus.GaugeValue, float64(query.Usage.UsedPublicMinutes*60))
 	metricChannel <- prometheus.MustNewConstMetric(c.currentBillingPeriodUsedSeats, prometheus.GaugeValue, float64(query.Usage.UsedSeats))
 
+	for _, state := range query.Metrics.StacksCountByState {
+		metricChannel <- prometheus.MustNewConstMetric(c.currentStacksCountByState, prometheus.GaugeValue, state.Value, state.Labels[0])
+	}
+
+	for _, state := range query.Metrics.ResourcesCountByDrift {
+		metricChannel <- prometheus.MustNewConstMetric(c.currentResourcesCountByDrift, prometheus.GaugeValue, state.Value, state.Labels[0])
+	}
+	
+	metricChannel <- prometheus.MustNewConstMetric(c.currentAvgStackSizeByResourceCount, prometheus.GaugeValue, float64(query.Metrics.AvgStackSizeByResourceCount[0].Value))
+	metricChannel <- prometheus.MustNewConstMetric(c.currentAverageRunDuration, prometheus.GaugeValue, float64(query.Metrics.AverageRunDuration[0].Value))
+	metricChannel <- prometheus.MustNewConstMetric(c.currentMedianRunDuration, prometheus.GaugeValue, float64(query.Metrics.MedianRunDuration[0].Value))
```

Comment:
Not sure if you need to cast to `float64` here. `dataPoint` values already are `float64`.

Reply:
fixed this one